src/Tooling/Bit.Tooling.Templates/TodoTemplate/Api/AppSettings.cs

```diff
@@ -8,11 +8,18 @@ public class AppSettings
 
         public EmailSettings EmailSettings { get; set; }
 
+        public HealCheckSettings HealCheckSettings { get; set; }
         public string UserProfileImagePath { get; set; }
 
         public string WebServerAddress { get; set; }
     }
 
+    public class HealCheckSettings
+    {
+        public bool EnableHealthChecks { get; set; }
+        public int MaximumMegabytesAllocated { get; set; }
```

Comment:
What's this maximum megabytes...?

Reply:
I renamed it to MaximumApplicationMemoryMegabytesAllocated. I think it is better to let the end-user decide about their application memory usage. also, 6 * 1024 * 1024 means 6 TB and it's very huge because unit it's megabyte